pkg/collector/collector.go

```diff
@@ -80,32 +85,69 @@ func (c Collector) StampDeployment(stack builder.Stack, config builder.Config, t
 	return err
 }
 
-func (c Collector) UpdateStatus(asg string, status string, updateFields map[string]string) error {
+func (c Collector) UpdateStatus(asg string, status string, updateFields map[string]interface{}) error {
+	Logger.Debugf("deployment statuses of previous autoscaling groups are started")
 	if err := c.MetricClient.DynamoDBService.UpdateRecord("deployment_status", asg, c.MetricConfig.Storage.Name, status, updateFields); err != nil {
 		return err
 	}
+	Logger.Debugf("deployment statuses of previous autoscaling groups are updated")
+
+	return nil
+}
+
+func (c Collector) UpdateStatistics(asg string, updateFields map[string]interface{}) error {
+	if err := c.MetricClient.DynamoDBService.UpdateStatistics(asg, c.MetricConfig.Storage.Name, updateFields); err != nil {
+		return err
+	}
 	Logger.Debugf("deployment metric is updated")
 
 	return nil
 }
 
-func (c Collector) GetAdditionalMetric(asg string) (map[string]string, error) {
+func (c Collector) GetAdditionalMetric(asg string, tgs []*string, logger *Logger.Logger) (map[string]interface{}, error) {
 	item, err := c.MetricClient.DynamoDBService.GetSingleItem(asg, c.MetricConfig.Storage.Name)
 	if err != nil {
 		return nil, err
 	}
 
-	ret := map[string]string{}
+	var baseTimeDuration float64
+	var startDate time.Time
+
+	curr := tool.GetKstTimestamp()
```

Comment:
I think It can be working with User Timezone that from configs in the future.

Reply:
Oh.. I think this should be included ASAP. Now goployer will save metrics based on KST. I think it is hard to support all timezone setting(too many timezones...)
How about using UTC only?

Reply:
UTC is good.
And If it can be override with time difference from UTC like 'UTC+9'. It will be helpful to users.

Reply:
Even though user wants to apply the different timezone, we need to ensure that it is okay to apply that timezone to all autoscaling groups of all region. If it is required to apply different timezones according to the region asg belongs to, then gathering metrics is impossible I think...

Reply:
We'd time to talk about this issue and we've got next solutions.

* Change metric unit `a day` to `an hour`.
* Metric working with UTC timezone only.